ee/clickhouse/process_event.py

```diff
@@ -183,6 +183,7 @@ def log_event(
     KafkaProducer().produce(
         topic=KAFKA_EVENTS_WAL,
         data={
+            "uuid": str(UUIDT()),  # UUID generated here specifically for idempotency with plugin server ingestion
```

Comment:
We should use the same UUID in process_event_ee right?

Reply:
Hm, it will be. But you're right, for now we won't be producing into Kafka with the plugin server, just consuming to allow for exporting, notifying etc. with plugins. So we need to a make this UUID common.